torch/_dynamo/variables/nn_module.py

```diff
@@ -231,6 +232,13 @@ def record_nn_module_stack():
                     arg = tx.pop()
                 return arg
             elif is_allowed(mod.__class__):
+                if nnmodule_has_hooks(
+                    mod, check_forward_hooks=True, check_backward_hooks=True
+                ):
+                    unimplemented(
+                        f"Forward/backward hooks aren't yet supported on 'allowed' modules (e.g. {mod.__class__}), "
+                        "which don't get traced through by dynamo. Graph-breaking to run hooks without compile."
+                    )
```

Comment:
The test failure comes from:
https://github.com/pytorch/pytorch/blob/1bcb88089468a6ebc667bd76256c4dd6f58b7ee3/test/dynamo/test_modules.py#L1070-L1071

Since ```LazyBatchNorm3d``` is 'allowed' modules, so triggered this graph break, where we expect no graph break in the unit test. To handle this, we have two options:
* Don't graph break if they are 'allowed' lazy modules, coz dynamo will eagerly initialize the lazy module later and remove these hooks.
* Allow graph breaks in the [unit test](https://github.com/pytorch/pytorch/blob/master/test/dynamo/test_modules.py#L1083).